src/main/java/com/palantir/gradle/versions/FixLegacyJavaConfigurationsPlugin.java

```diff
@@ -28,9 +28,9 @@
 import org.gradle.api.plugins.JavaPlugin;
 
 /**
- * This plugin exists in order to ensure versions in legacy java configurations (that are still
- * {@link Configuration#isCanBeResolved() resolvable} for compatibility reasons) can be resolved, and that they use
- * the locked versions.
+ * This plugin exists in order to ensure versions in legacy java configurations (that are still {@link
```

Comment:
I find something deeply unsettling about line splits in javadoc tags. It's totally irrational, and not worth thinking about...

Reply:
Do you dislike the concept or just the fact that it splits the `{@link`?

Reply:
I tend to optimize around putting the entire tag on a single line. Splitting on the space is as reasonable as anything, but I would prefer to avoid breaking them even if it leaves shorter lines before/after.

Reply:
No problem, I think I prefer that too. Made a tracking issue and we can address once the more glaring formatting problems are solved.